src/main/java/Event.java

```diff
@@ -0,0 +1,14 @@
+public class Event extends Task {
+    protected String start;
+    protected String end;
+    public Event(String description, int isDone, String start, String end) {
+        super(description, isDone);
+        this.start = start;
+        this.end = end;
+    }
+
+    public String toString() {
+        return "[E]" + "[" + getStatusIcon() + "] " + this.description + " (from: " + stringToDate(this.start) + " to: " +
```

Comment:
Line length should be no longer than 120 charecters. You can consider using linebreak